x-pack/plugins/lens/public/drag_drop/drag_drop.scss

```diff
@@ -52,3 +47,32 @@
     text-decoration: line-through;
   }
 }
+
+.lnsDragDrop-reorderableContainer {
+  position: relative;
+}
+
+.lnsDragDrop-reorderableDrop {
```

Comment:
```suggestion
.lnsDragDrop--reorderable {
```

Reply:
I've changed it to .lnsDragDrop__reorderableDrop because this is not a type of .lnsDragDrop but the 'shadow' element we can drop the reorderable elements to.